Release notes for .NET SDK 3.2.7

jeffrymorris opened the pull request.

jeffrymorris committed:
Release notes for .NET SDK 3.2.7

osfameron commented:
@jeffrymorris https://packages.couchbase.com/clients/net/3.2/Couchbase-Net-Client-3.2.7.zip is not yet resolving

osfameron commented on modules/project-docs/pages/sdk-release-notes.adoc:
```diff
@@ -107,6 +107,86 @@ git pull origin master
+
+Version 3.2.7 is the seventh release of the 3.2 series.
+
+https://packages.couchbase.com/clients/net/3.2/Couchbase-Net-Client-3.2.7.zip[Download] |
```
404

osfameron commented on modules/project-docs/pages/sdk-release-notes.adoc:
```diff
@@ -107,6 +107,86 @@ git pull origin master
+https://docs.couchbase.com/sdk-api/couchbase-net-client-3.2.7[API Reference] |
+https://www.nuget.org/packages/CouchbaseNetClient/3.2.7[Nuget]
+
+Note: as of 3.2.5 .NET Core 2.1 support has been dropped from the SDK. This corresponds to Microsoft's decision to EOL .NET Core 2.1 on August 21, 2021.
```
should this be `3.2.7` ?

osfameron commented on modules/project-docs/pages/sdk-release-notes.adoc:
```diff
@@ -107,6 +107,86 @@ git pull origin master
+Use TaskCreationOptions.RunContinuationsAsynchronously.
+
+* https://issues.couchbase.com/browse/NCBC-3101[NCBC-3101]: 
+Use System.Text.Json for bootstrap and internal K/V operations.
```
Are these 3x System.Text.Json items user-visible?

osfameron reviewed (changes requested):
The 3.2.5 / 3.2.7 comment needs resolving by either:

- fixing if typo
- giving more context (if late announcement) in 3.2.7 docs and also adding to 3.2.5 section below.

osfameron committed:
Gardening

 * removed non user-facing commits
 * minor wording fixes:
     past tense
     user visible fix
     (often the Gerrit title, motivation, or result is usable as-is)
 * removed HTML artefact

osfameron commented:
@jeffrymorris @RiPont remaining queries:

*  3.2.5/3.2.7 query re .Net 2.1 support
* 3.2.7.zip not resolving.
* (you might also want to review my changes, where I've deleted what I think are non-user-facing commits)

osfameron committed:
Clarify .Net Core 2.1 support drop version

This is since 3.2.5, but as we're announcing it during release
notes for 3.2.7, am adding it to all 3 versions 3.2.5, .6, .7
